lib/Doctrine/Migrations/AbstractMigration.php

```diff
@@ -73,11 +70,7 @@ public function warnIf(bool $condition, string $message = '') : void
 
         $message = $message ?: 'Unknown Reason';
 
-        $this->outputWriter->write(sprintf(
-            '    <comment>Warning during %s: %s</comment>',
-            $this->version->getExecutionState(),
-            $message
-        ));
+        $this->logger->warning($message, ['migration' => $this]);
```

Comment:
Since `$context` allows for more data, in order to keep following diffs smaller, what do you think about wrapping the `$context` argument?

```suggestion
        $this->logger->warning($message, [
            'migration' => $this,
        ]);
```

Reply:
this is currently managed by doctrine-coding standards, and wants it in-line

Reply:
That is correct - doctrine/cs forces single-line for arrays with a single element.